.github/DEVELOPMENT.md

```diff
@@ -25,6 +25,18 @@ With `-e` passed to `pip install` above pip can reference the code you are
 modifying in the *virtual env*. That way, you do not need to run `pip install`
 again to make your changes applied to the *virtual env*.
 
+### `pre-commit` checks
```

Comment:
What exact checks are run will change over time - doesn't seem as important.

Maybe we'd want to instead mention about `pre-commit install` at `Code style` header which already tells how to run these checks manually.

Reply:
Done